components/Dashboard/MyFeed/MyFeed.tsx

```diff
@@ -32,37 +32,20 @@ type Props = {
 const MyFeed: React.FC<Props> = ({ currentUser }) => {
   const { t } = useTranslation('my-feed')
 
-  /**
-   * Topic filter selection state
-   */
-  const [topic, setTopic] = useState('')
-
-  /**
-   * Language filter selection state
-   */
+  const [search, setSearchState] = useState('')
+  const [selectedTopicsFilters, setSelectedTopicsFilters] = useState<number[]>([])
 
   // Fetch languages that have at least 1 post
   const { data: languagesData } = useLanguagesQuery({
     variables: {
       hasPosts: true,
     },
   })
```

Comment:
We're going to need to update the `topics` query in `topics.ts` so that it uses the `hasPosts` argument to filter only topics that have existing posts and can then be used in the same way that we fetch only languages that have at least one post here in `useLanguagesQuery`. Currently we end up with just all topics in the dropdown including ones that have no posts